site/ko/blog/new-in-devtools-109/index.md

```diff
@@ -28,7 +28,7 @@ draft: true
 
 <!-- ## Recorder: Copy as options for steps, in-page replay, step’s context menu {: #recorder } -->
```

Comment:
@syang0624 you need to translate those line in comment. For example, you need to translate this whole line but keep the tag:


From
```
## Recorder: Copy as options for steps, in-page replay, step’s context menu {: #recorder }
```
to

```
## Replace with korean translation {: #recorder }
```

Reply:
@jecfish Hey Jecelyn, thanks for finding those that I missed! I will commit and mention you again once the translated version is updated :D